services/seller.js

```diff
@@ -8,25 +8,25 @@ const models = require('../models');
 module.exports = {
 
   createSeller: (seller) => {
-    const hash = bcrypt.hashSync(seller.password, config.bcryptSaltRounds);
-    seller.password = hash;
-
-    return models.seller.create(seller).then(seller => seller.dataValues);
+    return bcrypt.hash(seller.password, config.bcryptSaltRounds).then(hash => {
+      seller.password = hash;
+      return models.seller.create(seller).then(seller => seller.dataValues);
+    });
```

Comment:
```
createSeller: async (seller) => {
    const hash = await bcrypt.hash(seller.password, config.bcryptSaltRounds);
    const sellerDataValues = await models.seller.create(seller).dataValues;
    resolve(sellerdataValues);
}
```
I think this will be more neat, what do you think? (there might be syntax errors)